ensemble-view/src/main/java/hec/ensembleview/mappings/SingleValueComboBoxMap.java

```diff
@@ -13,26 +12,26 @@ public class SingleValueComboBoxMap {
     public final static Map<SingleValueSummaryType, List<List<Statistics>>> summaryStatisticsMap = new HashMap<>();
 
     static {
-        summaryComboBoxMap.put(SingleValueSummaryType.ComputeAcrossEnsembles, "Compute Across Ensembles for Each Time Step");
-        summaryComboBoxMap.put(SingleValueSummaryType.ComputeAcrossTime, "Compute Across Time Steps for Each Ensemble");
-        summaryComboBoxMap.put(SingleValueSummaryType.ComputeCumulative, "Computing Cumulative");
+        summaryComboBoxMap.put(SingleValueSummaryType.COMPUTEACROSSENSEMBLES, "Compute Across Ensembles for Each Time Step");
+        summaryComboBoxMap.put(SingleValueSummaryType.COMPUTEACROSSTIME, "Compute Across Time Steps for Each Ensemble");
+        summaryComboBoxMap.put(SingleValueSummaryType.COMPUTECUMULATIVE, "Computing Cumulative");
 
-        summaryStatisticsMap.put(SingleValueSummaryType.ComputeAcrossEnsembles,
+        summaryStatisticsMap.put(SingleValueSummaryType.COMPUTEACROSSENSEMBLES,
                 Arrays.asList(
-                        Arrays.asList(Statistics.MIN, Statistics.MAX, Statistics.AVERAGE, Statistics.MEDIAN, Statistics.PERCENTILE),
+                        Arrays.asList(Statistics.MIN, Statistics.MAX, Statistics.AVERAGE, Statistics.MEDIAN, Statistics.PERCENTILES),
                         Arrays.asList(Statistics.MIN, Statistics.MAX)
                 )
         );
-        summaryStatisticsMap.put(SingleValueSummaryType.ComputeAcrossTime,
+        summaryStatisticsMap.put(SingleValueSummaryType.COMPUTEACROSSTIME,
                 Arrays.asList(
                         Arrays.asList(Statistics.MIN, Statistics.MAX, Statistics.AVERAGE, Statistics.TOTAL, Statistics.MAXACCUMDURATION, Statistics.MAXAVERAGEDURATION),
-                        Arrays.asList(Statistics.MIN, Statistics.MAX, Statistics.AVERAGE, Statistics.MEDIAN, Statistics.STANDARDDEVIATION, Statistics.VARIANCE, Statistics.PERCENTILE)
+                        Arrays.asList(Statistics.MIN, Statistics.MAX, Statistics.AVERAGE, Statistics.MEDIAN, Statistics.STANDARDDEVIATION, Statistics.VARIANCE, Statistics.PERCENTILES)
                 )
         );
-        summaryStatisticsMap.put(SingleValueSummaryType.ComputeCumulative,
+        summaryStatisticsMap.put(SingleValueSummaryType.COMPUTECUMULATIVE,
                 Arrays.asList(
                         Arrays.asList(Statistics.CUMULATIVE),
```

Comment:
Minor delinting suggestion. Can change to List.of since there aren't multiple arguments.

Reply:
I did not see this suggestion from the linter.  How would I go about changing it to Listof?

Reply:
The below should work.
![image](https://github.com/HydrologicEngineeringCenter/FIRO_TSEnsembles/assets/50494918/2f945137-bcb3-4bf3-b1ae-fd4202b6f057)

Reply:
Strange, I can't seem to use the List.of but I could set the array as a singletonList.